api/models.py

```diff
@@ -258,6 +262,7 @@ class Message(models.Model):
     in_reply_to = HeaderFieldModel(blank=True, db_index=True)
     date = models.DateTimeField(db_index=True)
     last_reply_date = models.DateTimeField(db_index=True, null=True)
+    last_comment_date = models.DateTimeField(db_index=True, null=True)
```

Comment:
IIIRC the intention of `last_reply_date` is exactly for things such as `has:replies`, having both it and the new `last_comment_date`, and using the latter for `has:replies` is a bit messy, isn't it? How about just reusing or repurposing the existing field? I think it is okay to recalculate all `last_reply_date` fields while we do. (Is it possible to do that with a Django migration even if there is no DB schema change? If not, let's do a machenial rename?)

What do you think?

Reply:
Yeah, that was my first attempt. But the two are different:

* `last_reply_date` also includes the cover letter and patches, which is the right thing to do if you want to sort the series list so that newer series come first. But it is the wrong to do for `has:replies`, of course.

* `last_comment_date` does not include messages from the original author, which makes it a poor sorting order. I could have used simply a bool field (such as `has_comment`) but I decided for a date field instead to gain more flexibility in the future.

(And yeah, Django migrations are possible even without a DB schema change.  Here I'm adding the manually-written data migration in the same commit as the automatically-generated schema migration, but the two are completely independent).

Reply:
OK then! Could you update with some descriptions or comments to describe the fields? Then I'l merge the PR. Thanks.